src/components/App.js

```diff
@@ -1,8 +1,11 @@
 import React from 'react';
 import Display from './Display';
 import ButtonPanel from './ButtonPanel';
+import Calculate from '../logic/calculate';
 
 function App() {
+  const test = Calculate({ total: '5', next: 1, operator: '+' }, '=');
+  console.log(test);
```

Comment:
Awesome job testing your Calculate function, kindly remember to remove this, so that you won't get to resubmit for review because of these 2 lines.